### ISSUE TITLE
What is the working or goal of the Example Codes? 

### ISSUE THREAD

**Issue by dibakarbose** [pywinauto/pywinauto#484]
This is not answered properly and hence it is a difficulty to give a fix for Windows 10. Not all the codes will work for Windows 10.

If code behaviour is properly mentioned then people can try to give a fix for the same.

**Comment by dibakarbose** [comment]
Then I will start doing some changes and commit it. I will do the changes according to what I understand by reading the code.

Is that okay?

**Comment by vasily-v-ryabov** [comment]
Sure, pull requests are welcome. I think it's enough to make the code working on Win10 only. Of course, if you have an ability to maintain Win7/Win10 compatibility, it's even better, but not strictly necessary. Just put a comment at the beginning of every example script about OS requirements.

**Comment by vasily-v-ryabov** [comment]
Of course, new examples are welcome too. I would like to mention the target software should be available for free. Please provide download link in the initial comments if it makes sense.

**Comment by sahana8050** [comment]
Hi, I would like to work on updating and improving the example outputs and documentation for this issue. Please assign it to me if possible.
